pkg/machine/config.go

```diff
@@ -111,10 +114,46 @@ func (rc RemoteConnectionType) MakeSSHURL(host, path, port, userName string) url
 	return uri
 }
 
+func GetDataHome() (string, error) {
+	if runtime.GOOS == "windows" {
+		home, err := os.UserHomeDir()
+		if err != nil {
+			return "", err
+		}
+		return filepath.Join(home, ".local", ".share"), nil
+	}
+
+	return homedir.GetDataHome()
+}
+
+func GetUserHome() (string, error) {
+	if runtime.GOOS == "windows" {
+		home, err := os.UserHomeDir()
+		if err != nil {
+			return "", err
+		}
+		return home, nil
+	}
+
+	return homedir.Get(), nil
```

Comment:
Why not fix homedir to work correctly on Windows?

Reply:
Is this wrong?
// Get returns the home directory of the current user with the help of
// environment variables depending on the target operating system.
// Returned path should be used with "path/filepath" to form new paths.
func Get() string {
	return os.Getenv(Key())
}